design/definitions.go

```diff
@@ -236,6 +236,8 @@ type (
 		QueryParams *AttributeDefinition
 		// Payload blueprint (request body) if any
 		Payload *UserTypeDefinition
+		// PayloadOptional describes Payload is optional or not
```

Comment:
Replace with `PayloadOptional is true if the request payload is optional, false otherwise.`